environment.yml

```diff
@@ -61,3 +61,5 @@ dependencies:
         - pylru==1.0.9
         - hyperopt
         - polling
+        - pybullet
```

Comment:
Please always tag a version for major dependencies.

Reply:
Okay. I'll add that.